web/src/user_group_pill.js

```diff
@@ -76,3 +77,13 @@ export function typeahead_source(pill_widget) {
     const groups = user_groups.get_realm_user_groups();
     return filter_taken_groups(groups, pill_widget);
 }
+
+export function create_pills($pill_container, pill_config) {
```

Comment:
Can this be moved to its own commit or to the commit which uses the function?

Reply:
Yes, moved to the commit using this function.